config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -61,6 +61,12 @@ config CHIP_MALLOC_SYS_HEAP_SIZE
 config NVS_LOOKUP_CACHE_SIZE
     default 512
 
+# See config/zephyr/Kconfig for full definition
+config CHIP_FACTORY_RESET_ERASE_NVS
```

Comment:
Why is this moved? It could be kept in the previous place even if the condition on `!CHIP_CRYPTO_PSA` was needed.

Reply:
Oh, Kamil asked for that :). So it was overriden in `nrfconnect/.../Kconfig` only because Telink had a different default but this setting was still part of the Matter Zephyr module, so it seemed OK to put define it there instead of in `Kconfig.defaults` but whatever :).

Reply:
Maybe let's keep it in Kconfig as it was, but we need to add the dependency to PSA Crypto until we have a better way to keep keys alive after performing the factory reset.